component-runtime-impl/src/test/java/org/talend/sdk/component/runtime/record/RecordBuilderImplTest.java

```diff
@@ -56,6 +57,11 @@ void providedSchemaGetSchema() {
                         .build())
                 .build();
         assertEquals(schema, new RecordImpl.BuilderImpl(schema).withString("name", "ok").build().getSchema());
+
+        Schema.EntriesOrder e = new Schema.EntriesOrder(new RecordImpl.BuilderImpl().getCurrentEntries()
+                .stream()
+                .map(Schema.Entry::getName)
+                .collect(Collectors.toList()));
```

Comment:
did not catch, but what's the purpose?

Reply:
it appears after rebase from master, and with compile error; i just fix the error without asking purpose. I think it can be deleted.